docs_italia_convertitore_web/static/css/docs_italia_convertitore.css

```diff
@@ -17,3 +17,6 @@
     cursor: pointer;
     margin-top: -8px;
 }
+.dropzone .dz-preview .dz-error-message {
+    top: 150px !important;
```

Comment:
Come mai hai dovuto modificare questo?

Reply:
Perchè Dropzone aggiunge il pulsante di rimozione dei file in basso e se hai un errore il popup ti nasconde il pulsante di rimozione.

Allego stackoverflow (ci sono anche gli screenshots)
https://stackoverflow.com/questions/28691280/dropzone-js-remove-link-hidden-behind-pop-up-message

Reply:
Oh ok! C'è anche una [issue](https://github.com/enyo/dropzone/issues/1510) a riguardo, mai risolta a dovere. :)